docs/api.md

```diff
@@ -406,6 +406,12 @@ if the url has a query parameter of `keys=true`.
   
   <!--end-request-body-post-accountcreate-metricsContext-->
 
+* `marketingOptIn`: *boolean*
+
+  <!--begin-request-body-post-accountcreate-marketingOptIn-->
+  
```

Comment:
Yeah, that is kind of weird. Happy to do something different if people have better suggestions.